service/history/shard/controller_impl.go

```diff
@@ -117,17 +117,16 @@ func (c *ControllerImpl) Start() {
 	c.contextTaggedLogger = log.With(c.logger, tag.ComponentShardController, tag.Address(hostIdentity))
 	c.throttledLogger = log.With(c.throttledLogger, tag.ComponentShardController, tag.Address(hostIdentity))
 
-	c.acquireShards()
```

Comment:
What happened to this call?

Reply:
It's intentionally removed. Calling `acquireShards` requires asking membership what shards this history instance should own. With this change, the instance hasn't joined membership when this shard controller Start method is called, so it can't call acquireShards here anymore. The shard controller will get a membership update immediately after the instance joins, though, and that will call acquireShards.